tamr_unify_client/project/attribute_mapping_collection.py

```
@@ -0,0 +1,25 @@
+import requests
+
+from tamr_unify_client.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection:
+    def __init__(self, map_url):
+        self.map_url = map_url
+
+    def stream(self):
+        all_maps = requests.get(self.map_url).json()
+        for mapping in all_maps:
+            yield AttributeMapping(mapping)
+
+    def by_resource_id(self, resource_id):
+        maps = self.stream()
+        for mapping in maps:
+            str_id = str(mapping.relative_id)
+            split_id = str_id.split("attributeMappings/")[1]
+            if resource_id == split_id:
+                return mapping
+
```

If you don't find it, you'll want to raise an error